pkgs/tools/security/gnupg/20.nix

```diff
@@ -3,13 +3,13 @@
 
 # Each of the dependencies below are optional.
 # Gnupg can be built without them at the cost of reduced functionality.
-, pinentry ? null, x11Support ? true
+, pinentry ? null, guiSupport ? true
```

Comment:
@oxij I’m afraid, I can’t agree with your judgement.

There are 13 packages having the `x11Support` flag: w3m, mplayer, mpv, gcc, clisp, python, SDL, dbus, ghostscript, conky, pulseaudio, gnupg, pass.

* w3m has a separate `graphicsSupport` flag and uses `x11Support` exactly to pull libX11.
* mplayer and mpv, understandably, use this flag to pull libX11.
* I’m not sure about gcc, this seems to be related to the java compiler and AWT, but it also pulls xlibs.
* clisp pulls libX11, libXau and libXt.
* Python apparently uses libX11 and xlibsWrapper
* SDL, pretty clear.
* dbus and ghostscript: no idea why, but they pull various X libraries
* conky actively uses X
* pulseaudio, no idea, but it uses xlibsWrapper, libXtst and libXi
* gnupg (we are here)
* pass uses xclip

On the other hand, there are 4 packages (redshift, qbittorrent, git, mercurial) that use the `guiSupport` flag.

Therefore, out of 13 packages there are around 10 that use this flag to control support for the _actual X server_, not GUI in general, and 2 or 3 whose flag should probably be renamed.

Anyway, I wouldn’t call a convention followed by 13 packages “standard”.